src/frontend/src/instance.rs

```diff
@@ -494,6 +494,10 @@ pub fn check_permission(
         Statement::CreateExternalTable(stmt) => {
             validate_param(&stmt.name, query_ctx)?;
         }
+        Statement::CreateFlowTask(stmt) => {
```

Comment:
CreateFlow ?

Reply:
Done. Others are from proto file, going to change them afterward